### ISSUE TITLE
Stuck on "Building Course Data"

### ISSUE THREAD

**Issue by heliomarpm** [heliomarpm/udemy-downloader-gui#49]
### Udeler | Udemy Course Downloader

1.11.1

### Running as Business Account

- [ ] Yes

### Steps to reproduce

Sorry Guys, I can't reproduce this problem.
In version 1.11.2 I added a timeout to try to throw an error message in the logs section and hope to find some clues, but as I said in my tests I managed to start downloading all these courses.
Maybe the problem is related to some firewall or dns of your internet connection

Centering the issues 
- #12 
- #27 

### ✔️ Expected Behavior

![OK](https://user-images.githubusercontent.com/13087389/158081480-49a6fff4-2f54-42bd-8cc1-05e65471692f.png)

### ❌ Actual Behavior

![NOK](https://user-images.githubusercontent.com/28645933/158109366-3889fe78-7005-466f-bf90-9b29a22c0597.png)

**Comment by mabdulmoez** [comment]
I have the same issue.

**Comment by cwichel** [comment]
I've kind of found a temporary fix... 
- Downgrade to version 1.9.0 (started working again in my case). 
- Upgraded one release at a time up to 1.11.2 (testing on each one). 

It's not clean nor pretty but worked for me...

**Comment by freddygarcia** [comment]
I noticed that by default the app (v1.11.2) stucks on Building Course Data.
![image](https://user-images.githubusercontent.com/6600991/174388267-cbd55c64-cf7c-47f4-8fea-aaf894091c4f.png)

Taking a look to the settings, the download folder path seems to be wrong .

_C:\Users\Freddie/Downloads instead of C:\Users\Freddie\Downloads_

![image](https://user-images.githubusercontent.com/6600991/174388340-afe48ed8-7e66-4a76-a70f-733b318bc57b.png)

After select the Download folder, it working fine.

Make sure you save the changes. Since the save button is hidden in the bottom of the screen, I had to guess that I need to click it. 🤣 

![image](https://user-images.githubusercontent.com/6600991/174388491-4415bdb2-f0e9-4f5a-b753-c8c4e2627584.png)

![image](https://user-images.githubusercontent.com/6600991/174388769-2bb73426-7ef6-4a29-aaa4-be15a454fe8f.png)
